index.js

```diff
@@ -288,6 +288,8 @@ AFRAME.registerComponent('super-hands', {
     const hoverEls = this.hoverEls
     const hitElIndex = this.hoverEls.indexOf(hitEl)
     let hoverNeedsUpdate = false
+    if (dist && intersection.instanceId !== undefined)
+      hitEl.object3D.userData = { instanceId: intersection.instanceId };
```

Comment:
Where is this instanceId coming from? When I do a search I found it nowhere. I'm not super familiar with the super-hands code base and what it uses for the collision detection.

Reply:
I see yes, it's threejs raycaster that returns instanceId of the intersection on an InstancedMesh.
https://threejs.org/examples/?q=rayc#webgl_instancing_raycast